package/Dockerfile

```diff
@@ -7,11 +7,12 @@ RUN zypper --non-interactive update \
     && zypper --non-interactive install \
     git
 
-RUN git clone -b v0.6.8 --depth 1 https://github.com/aquasecurity/kube-bench.git
+RUN git clone -b v0.6.11 --depth 1 https://github.com/aquasecurity/kube-bench.git
 
 FROM registry.suse.com/bci/bci-base:15.4
-ARG kube_bench_tag=0.6.8
-ARG sonobuoy_version=0.56.7
+ARG kube_bench_tag=0.6.11
```

Comment:
Question: line 10 is cloning the same version of kube-bench. Could we re-order the lines to avoid having the version hardcoded in one place and specified as variable in another?

Reply:
Good catch! I moved the ARG block to the top as well, as it makes it clearer as to the configurable bits.

Reply:
Making some additional changes as this won't work due to the multi-staged image approach.